http-api/src/lib.rs

```diff
@@ -71,6 +85,15 @@ pub async fn run(options: Options) {
     }
 }
 
+/// Return the peer id for the node identity.
+/// `GET /v1/peer`
+async fn get_peer_id(peer_id: PeerId) -> Result<impl warp::Reply, warp::Rejection> {
+    let response = json!({
+        "id": peer_id.to_string(),
+    });
+    Ok(warp::reply::json(&response))
+}
+
```

Comment:
Can we bring some consistency to the names of things?

peer_id_route -> peer
get_peer_id -> peer_handler

Reply:
commit 801ffec renames.